site/content/en/docs/products/community/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'CVAT Community'
+linkTitle: 'CVAT Community'
+weight: 3
+description: 'Documentation for the self-hosted open-source edition.'
+---
+
+CVAT Community is the free, self-hosted open-source edition of CVAT,
+suitable for technical teams that prefer full control over their infrastructure and custom deployments.
+
+Below are the documents relevant specifically to the **CVAT Community** edition:
```

Comment:
The empty list below looks a little bit underwhelming, can we make this line conditional maybe?